src/TextInput/TextInput.tsx

```diff
@@ -228,7 +228,16 @@ export class TextInput extends React.Component<Props, State> {
           value={this.props.value as any}
         />
         {this.props.enableShow && (
-          <button type="button" className="TextInput--link" onClick={this.toggleHidden}>
+          <button
+            type="button"
+            className="TextInput--link"
+            onClick={this.toggleHidden}
+            onMouseDown={() => {
```

Comment:
`onMouseDown` should give you access to an event `e` param. It's probably preferable to use that. (Right now, we're making use of the global `event` variable on the `window` object.) The change would look like this:

```jsx
onMouseDown={e => {
  e.preventDefault();
  e.stopPropagation();
}}
```

Reply:
Yea. Copy paste error on my part -.-